modules/azure/cosmosdb/main.tf

```
@@ -0,0 +1,23 @@
+resource "azurerm_cosmosdb_account" "db" {
+  name                = "${local.network_name}-cosmos-db"
```

```suggestion
  name                = "${local.network_name}-cosmosdb"
```
for consistency with the resource name.